client/src/pages/ProfileListings/ProfileListings.tsx

```diff
@@ -54,6 +54,7 @@ export default function ProfileListings(): JSX.Element {
 
   const initializeProfiles = () => {
     getProfiles().then((data) => {
+      console.log(data);
```

Comment:
This can be removed, we don't need to console log this in production

Reply:
Sure, I do that